UKHomeOffice/sgmr-service
# SPCR-89 Remove redux as it is not used

## Description
Redux and associated packages removed as we are not using redux/redux sagas

## To test

- delete your `pacakge-lock.json` and `node-modules`
- run `npm i`
- start the app locally
- the app should load and run


## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
